### PR TITLE
Document how to try out and debug builds locally

### PR DESCRIPTION
**Pull request by chmeliik**
CLOUDBLD-5518

### CODE REVIEW

File: docs/using_requests_locally.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+```shell
+cachito-download.sh https://cachito.example.org/api/v1/requests/1 /tmp/cachito-1
+```
+
+## Use the right environment
+
+If you are debugging a build, make sure to match the target environment as closely as
+possible. For example, if you are building a container based on `python:3.9`, then you
+may want to try out the build it that base image.
```
Comment (akhmanova) [inline docs/using_requests_locally.md:24]: @chmeliik Could you please clarify/rewrite the last part of the sentence? 

 `build it that base image`

File: docs/using_requests_locally.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+```shell
+cd /tmp/cachito-1/remote-source
+podman run --rm -ti -v "$PWD:$PWD:z" python:3.9 bash
```
Comment (akhmanova) [inline docs/using_requests_locally.md:28]: Why do you suggest using the current directory as a volume?

**Review by akhmanova — CHANGES_REQUESTED** [review]

File: docs/using_requests_locally.md (hunk 1 above)
Comment (chmeliik) [reply docs/using_requests_locally.md]: Oh, that's just a typo :smile: 
Does it make more sense now?

File: docs/using_requests_locally.md (hunk 2 above)
Comment (chmeliik) [reply docs/using_requests_locally.md:29]: Added comment

File: docs/using_requests_locally.md (hunk 1 above)
Comment (akhmanova) [reply docs/using_requests_locally.md]: Sure! Small English words can bring no sense sometimes :D

File: docs/using_requests_locally.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
+# Using Cachito requests locally
```
Comment (akhmanova) [inline docs/using_requests_locally.md:1]: @chmeliik do we want to include a link to the file into the main README file?

Comment (chmeliik) [reply docs/using_requests_locally.md:1]: Good idea! Will do.

**Review by akhmanova — APPROVED** [review]
Nice work! LGTM. 

I really admire your ability to speak about engineering solutions with human language. :D

File: docs/using_requests_locally.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+[OSBS][osbs-cachito]. Installing the content from a finished request without the help of
+OSBS may seem like a daunting task, but is in fact not that difficult. Doing so may be
+especially useful when trying to debug a build failure without going through the entire
+process again.
+
+## Get the relevant files
+
+A Cachito request has three main parts: the **archive**, the **environment variables**
+and the **configuration files**. Getting these manually is a bit of a pain, which is
+why we use the [cachito-download.sh](../bin/cachito-download.sh) script to do it. You
```
Comment (lcarva) [inline docs/using_requests_locally.md:13]: I didn't even know this was a thing!

**Pull request merged**